plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectConfigTest.java

```diff
@@ -34,35 +34,37 @@
 public class AppEngineProjectConfigTest {
 
   private AppEngineProjectConfig config = new AppEngineProjectConfig();
-  
+
+  @Test
   public void testProject() throws CoreException {
     try {
       IWorkspace workspace = ResourcesPlugin.getWorkspace();
       IProject project = workspace.getRoot().getProject("foobar");
+      project.create(null);
       config.setProject(project);
-      Assert.assertTrue(project.getRawLocationURI().getPath().endsWith("foobar"));
+      Assert.assertTrue(project.getLocationURI().getPath().endsWith("foobar"));
```

Comment:
Not sure honestly. I don't really get what this test case wants to achieve. When this test was enabled by adding `@Test`, it was failing, as `getRawLocationURI` was returning `null`. `getLocationURI` doesn't at least return `null`, and it actually makes the test pass.